scripts/time-drift.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+#
+# zenoss-inspector-tags verify
+#
+
```

Comment:
Z has recommended based on several instances where ntp wasn't working or the time drift was too far apart between host, which was causing issues. I am wanting to run this against all the systems in the environment.